From 4fc3a18ab7c46e4bc375fa3ce59b7fb1b173f35b Mon Sep 17 00:00:00 2001
From: Pavel Emelyanov <xemul@openvz.org>
Date: Tue, 9 Sep 2008 19:25:00 +0400
Subject: [PATCH] cpt: bump image version to VERSION_26

Images from older kernels will most likely not get restored...

Signed-off-by: Pavel Emelyanov <xemul@openvz.org>
---
 kernel/cpt/cpt_context.c |    2 +-
 kernel/cpt/rst_context.c |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/kernel/cpt/cpt_context.c b/kernel/cpt/cpt_context.c
index 58a8069..e3f3f9f 100644
--- a/kernel/cpt/cpt_context.c
+++ b/kernel/cpt/cpt_context.c
@@ -122,7 +122,7 @@ int cpt_major_hdr_out(struct cpt_context *ctx)
 	hdr.cpt_signature[2] = CPT_SIGNATURE2;
 	hdr.cpt_signature[3] = CPT_SIGNATURE3;
 	hdr.cpt_hdrlen = sizeof(hdr);
-	hdr.cpt_image_version = CPT_VERSION_20;
+	hdr.cpt_image_version = CPT_VERSION_26;
 #ifdef CONFIG_X86_64
 	hdr.cpt_os_arch = CPT_OS_ARCH_EMT64;
 #elif defined(CONFIG_X86_32)
diff --git a/kernel/cpt/rst_context.c b/kernel/cpt/rst_context.c
index 47e4f35..cc381fe 100644
--- a/kernel/cpt/rst_context.c
+++ b/kernel/cpt/rst_context.c
@@ -178,7 +178,7 @@ int rst_open_dumpfile(struct cpt_context *ctx)
 	ctx->start_time.tv_nsec = h.cpt_start_nsec;
 	ctx->kernel_config_flags = h.cpt_kernel_config[0];
 	ctx->iptables_mask = h.cpt_iptables_mask;
-	if (h.cpt_image_version > CPT_VERSION_20 ||
+	if (h.cpt_image_version > CPT_VERSION_26 ||
 			CPT_VERSION_MINOR(h.cpt_image_version) > 1) {
 		eprintk_ctx("Unknown image version: %x. Can't restore.\n",
 				h.cpt_image_version);
-- 
1.6.0.6

